apps/web/src/utils/usernames.ts

```diff
@@ -583,6 +583,28 @@ export async function getBasenameOwner(username: BaseName) {
   } catch (error) {}
 }
 
+export function buildBasenameNameExpiresContract(username: BaseName): ContractFunctionParameters {
```

Comment:
nit: it looks like this isn't used elsewhere--should prob inline into getBasenameNameExpires, or at least not export